codegen-renderers/src/test/kotlin/io/vrap/codegen/languages/TestCodeGenerator.kt

```diff
@@ -42,6 +45,37 @@ class TestCodeGenerator {
         generatorComponent.generateFiles()
     }
 
+    @Test
+    fun generateJavaModelsSecond() {
+        val generatorConfig = CodeGeneratorConfig(basePackageName = "com.commercetools.importer", outputFolder = Paths.get("build/gensrc/java"))
+        val generatorModule = GeneratorModule(apiProvider, generatorConfig, JavaBaseTypes)
+        val generatorComponent = GeneratorComponent(generatorModule, JavaModelModuleSecond())
+        generatorComponent.generateFiles()
+    }
+
+    /**
+     * This test method uses code generator to generate interface and a class for simple-type.raml which is a part of the test-api.raml located in the resources
+     * folder. After the classes are generated, it checks if they are the same as the ones specified in SimpleType.txt and SimpleTypeImpl.txt.
+     */
+    @Test
+    fun compareGenerated() {
```

Comment:
I like this test, but I wouls suggest to rename it to sth. that expresses what you wrote in the comment. If you run this test and it fails, the test runner will only show the method name and not the comment above it ;-)